client/state/notices/middleware.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { truncate } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { successNotice, errorNotice } from 'state/notices/actions';
+import { getSitePost } from 'state/posts/selectors';
+import {
+	POST_DELETE_FAILURE,
+	POST_DELETE_SUCCESS,
+	POST_SAVE_SUCCESS
+} from 'state/action-types';
+
+export const OBSERVERS = {
```

Comment:
[Previous discussion](https://github.com/Automattic/wp-calypso/pull/6013#discussion_r68981097).  After thinking about it some more I like `PascalCase` the best for situations like this, but we should document that in a separate PR.  What are the applicable situations?  Collections of objects/constants/functions?  The pattern is almost like a class with static-only methods in languages that support such.